apps/opencs/model/world/resourcesmanager.cpp

```diff
@@ -31,11 +31,26 @@ void CSMWorld::ResourcesManager::setVFS(const VFS::Manager *vfs)
     addResources (Resources (vfs, "videos", UniversalId::Type_Video));
 }
 
-const VFS::Manager* CSMWorld::ResourcesManager::getVFS() const
+VFS::Manager* CSMWorld::ResourcesManager::getVFS() const
```

Comment:
Access to a non const version of the VFS manager is needed in the Data class to rebuild the indices. I could alternately pass it directly to the Data class from the Document class in the constructor.